docs/user_guide/productionizing/configuring_access_to_gpus.md

```diff
@@ -6,31 +6,405 @@
+If your infrastructure requires additional tolerations for the scheduling of GPU resources to succeed, adjust the following section in the Helm values file:
+
+**flyte-core**
+```yaml
+configmap:
+  k8s:
+    plugins:
+      k8s:
+        resource-tolerations:
+        - nvidia.com/gpu: 
+          - key: "mykey"
+            operator: "Equal"
+            value: "myvalue"
+            effect: "NoSchedule"  
+```
+**flyte-binary**
+```yaml
+configuration:
+  inline:
+    plugins:
+      k8s:
+        resource-tolerations:
+        - nvidia.com/gpu: 
+          - key: "mykey"
+            operator: "Equal"
+            value: "myvalue"
+            effect: "NoSchedule" 
+```
+>For the above configuration, your worker nodes should have a  `mykey=myvalue:NoSchedule` configured [taint](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/).
+
+## Requesting a specific GPU device
+
+In this example, the goal is to run the task on a specific type of accelerator: NVIDIA Tesla V100 :
+
+
+```python
+from flytekit import ImageSpec, Resources, task
+from flytekit.extras.accelerators import V100
+
+image = ImageSpec(
+    base_image= "ghcr.io/flyteorg/flytekit:py3.10-1.10.2",
+     name="pytorch",
+     python_version="3.10",
+     packages=["torch"],
+     builder="envd",
+     registry="<YOUR_CONTAINER_REGISTRY>",
+ )
+
+@task(requests=Resources( gpu="1"),
+              accelerator=V100, 
+              ) #NVIDIA Tesla V100
+def gpu_available() -> bool:
+   return torch.cuda.is_available()
+```
+
+
+### How it works?
+
+When this task is evaluated, `flytepropeller` injects both a toleration and a [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) for a more flexible scheduling configuration.
+
+An example pod spec on GKE would include the following:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: cloud.google.com/gke-accelerator
+            operator: In
+            values:
+            - nvidia-tesla-v100
+  containers:
+  - resources:
+      limits:
+        nvidia.com/gpu: 1
+  tolerations:
+  - key: nvidia.com/gpu  # auto
+    operator: Equal
+    value: present
+    effect: NoSchedule
+  - key: cloud.google.com/gke-accelerator
+    operator: Equal
+    value: nvidia-tesla-v100
+    effect: NoSchedule
+```
+### Configuring the nodeSelector
+The `key` that the injected node selector uses corresponds to an arbitrary label that your Kubernetes worker nodes should already have. In the above example it's `cloud.google.com/gke-accelerator` but, depending on your cloud provider it could be any other value. You can inform Flyte about the labels your worker nodes use by adjusting the Helm values:
```

Comment:
```suggestion
The `key` that the injected node selector uses corresponds to an arbitrary label that your Kubernetes worker nodes should already have. In the above example, it's `cloud.google.com/gke-accelerator` but, depending on your cloud provider it could be any other value. You can inform Flyte about the labels your worker nodes use by adjusting the Helm values:
```